src/Error404.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './App.css'
+
+function Error404() {
+    return (
+      <div>
+        <body class="rails-default-error-page">

```

Comment:
Why are you using body tag inside div tag

Reply:
removed